ChangeLog.d/fix_some_resource_leaks.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+   * Fixed resource leaks in mbedtls_pk_parse_public_key() under
+     certain error conditions.
```

Comment:
We should clarify what the error conditions are. Changelog entries are for users, and users need to know how the bug can potentially be triggered. For example, can it be triggered by a remote entity sending bad data? Or is it typically only caused by a misconfiguration in the application? etc.

```suggestion
   * Fixed resource leaks in mbedtls_pk_parse_public_key() in low memory
     conditions.
```